apps/designer-standalone/src/environments/armToken.json

```diff
@@ -1 +0,0 @@
-{}
```

Comment:
@hartra344 I know we have the pre-commit to prevent any token from ever populating this, but do we actually need this file? Seems like it can be created when generating ARM token, and by having this file deleted we prevent git from tracking, as the file is already in the repo

Reply:
ah nevermind, i see why the file is needed when we grab the token in the environment.ts file

Reply:
yep, that's the reason haha, otherwise it required async loading stuff which would be flakey when the app first started. This was the easiest, most reliable way to do it